File: 816-design-hashset/design-hashset.js

var MyHashSet = function() {
    this.hashSet = new Set()
};

/** 
 * @param {number} key
 * @return {void}
 */
MyHashSet.prototype.add = function(key) {
    this.hashSet.add(key)
};

/** 
 * @param {number} key
 * @return {void}
 */
MyHashSet.prototype.remove = function(key) {
    if(this.hashSet && this.hashSet.has(key)){
        this.hashSet.delete(key)
    }
};

/** 
 * @param {number} key
 * @return {boolean}
 */
MyHashSet.prototype.contains = function(key) {
    return this.hashSet.has(key)
};

/** 
 * Your MyHashSet object will be instantiated and called as such:
 * var obj = new MyHashSet()
 * obj.add(key)
 * obj.remove(key)
 * var param_3 = obj.contains(key)
 */

File: 816-design-hashset/design-hashset.py
class MyHashSet:
    
    def __init__(self):
        self.hashSet = set()
    def add(self, key: int) -> None:
        self.hashSet.add(key)
    def remove(self, key: int) -> None:
        if key in self.hashSet and self.hashSet :
            self.hashSet.remove(key)
    def contains(self, key: int) -> bool:
        return key in self.hashSet

# Your MyHashSet object will be instantiated and called as such:
# obj = MyHashSet()
# obj.add(key)
# obj.remove(key)
# param_3 = obj.contains(key)